enum.c

```diff
@@ -3369,13 +3370,19 @@ drop_i(RB_BLOCK_CALL_FUNC_ARGLIST(i, args))
 
 /*
  *  call-seq:
- *     enum.drop(n)               -> array
+ *    drop(n) -> array
+ *
+ *  For positive integer +n+, returns the last +n+ elements:
  *
- *  Drops first n elements from <i>enum</i>, and returns rest elements
- *  in an array.
+ *    r = (1..4)
+ *    r.drop(2) # => [3, 4]
+ *
+ *    h = {foo:0, bar: 1, baz: 2, bat: 3}
+ *    h.drop(2) # => [[:baz, 2], [:bat, 3]]
  *
- *     a = [1, 2, 3, 4, 5, 0]
- *     a.drop(3)             #=> [4, 5, 0]
+ *  For <tt>n == 0</tt>, returns all elements:
```

Comment:
You can keep an example of `0`, but you should remove the separate explanation, since it is the same behavior.

Reply:
Done.

Reply:
Should change the `:` at the end to `.`.